index.html

```diff
@@ -0,0 +1,122 @@
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Art Exhibition: Home page</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-light bgp-light">
+      <div class="container-fluid">
+        <a class="navbar-brand p-0 m-0" href="#"><img class="p-0 m-0 logo-container"
+            src="./img/log.jpg" alt="icon"></a>
+        <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page"
+                href="./about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">News</a>
+            </li>
+            <li class="nav-item mx-md-5 nav-content">
+              <a class="nav-link">CC Campain</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="body-bg d-flex justify-content-center">
+      <div class="header">
+        <div class="title d-flex justify-content-center flex-column">
+          <h4>"Hello! Sharing world"</h4>
+          <h1>ART EXHIBITION - 2022</h1>
+          <div class="descruption">
+            <p>Art exhibitions represent the gathering of art objects into a space for a temporary event. The exhibition or show may include work by a single artist, art works on a single topic or theme, from a specific era, in a defined medium, from a defined geographic region, or a combination of all of these.</p>
+          </div>
+          <div class="prod-year">
+            <h3>
+              2022.08.15 (Monday) ~ 16(Tuesday)
+            </h3>
+            <p>@informations by Mufungizi Gloire</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <div class="container-programs">
+    <div class="main-program-container container-fluid text-light programs-class">
+      <h1 class="text-center my-2">Main Programs</h1>
+      <div class="container d-flex my-3 justify-content-center line-bellow"></div>
+    </div>
+  </div>
+  <div class="ranking-section container-fluid py-5">
+    <h2 class="container text-cener pt-5 d-flex justify-content-center">Feature Artists</h2>
```

Comment:
- [ ] I think it would be better to use HTML5 semantic tag `<section>` over `divs` tags , To follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 👌

For example, instead of `<div class="ranking-section>`  I suggest `<section class="ranking-section"> ` 👌

- [ ] Please implement this change in all sections you have in the home and about pages eg ( main programs, feature artists, partners, section-product, rank-section-about, ... )

Reply:
Thank you for your feedback :memo:
Changes are made with the previous corrections.
![image](https://user-images.githubusercontent.com/37586807/181683345-a61f404b-7e5e-40fc-935c-a2a875ef584c.png)